site/profile/manifests/cvmfs.pp

```diff
@@ -22,19 +25,26 @@
     require => [Package['cvmfs-repo'], Package['cc-cvmfs-repo']]
   }
 
+  if 'ref.mugqic' in $repositories {
+    file { '/etc/cvmfs/config.d/ref.mugqic.conf':
```

Comment:
Preferably any local customizations/modifications should be in .local files, especially if the list of repositories using an alien cache could be extended, we would not want to overwrite other pre-existing config files.  
So it should be /etc/cvmfs/config.d/ref.mugqic.local
(Although I'm not sure if this would collide with what the CVMFS puppet module itself does.)